pkg/infra/game/agent/random.go

```diff
@@ -0,0 +1,23 @@
+package agent
```

Comment:
This is an initial sample of what the Agent will look like for the MVP. At the moment there is no peer to peer communication and the only decision the agents need to take is to whether to fight or not. This is what the HandleFight function is for.